### ISSUE TITLE
[BUG] App does Launch but the GUI seems broken

### ISSUE THREAD

**Issue by LL187** [radj307/volume-control#107]
### Version

6.3.0

### Windows Version

Windows 11 Pro Version 22H2 (Build 22621.2428)

### Actual Behaviour

The Program Succesfully installs and even runs in the Taskbar hoever the GUI seems broken? I tried reinstalls but always ended up with the same result of Volume Control Running in the background, but i'm unable to get the GUI to show up.

![Desktop Screenshot 2023 10 25 - 08 25 42 41](https://github.com/radj307/volume-control/assets/34402392/f4b48d3c-fbf5-45fe-bb5d-1de3bbbcebca)

The Broken GUI, simply not showing a Window.

![Desktop Screenshot 2023 10 25 - 08 26 37 06](https://github.com/radj307/volume-control/assets/34402392/e76697e3-5b7f-4553-827e-52a1771660f7)

Software seems to Run normally.

![Desktop Screenshot 2023 10 25 - 08 26 44 67](https://github.com/radj307/volume-control/assets/34402392/e05c4692-b1da-463e-9cd3-0d091993371a)

and even lets me control it here.

I'm not sure if the Screenshots are of any help, but i am little puzzled of what could have went wrong, i also went on search to find a log but couldn't find one either.

### Reproduction Steps

1. Install Volume Control either using winget or the installer
2. Run Like normal
3. go to the Taskbar symbol right click and select show

**Comment by radj307** [comment]
The log you posted is a good start but the log filter doesn't include `DEBUG` or `TRACE` messages, which I will need in order to debug this. To include them, do the following steps:  
1. Open `VolumeControl.json` in a text editor
2. Find `"LogFilter": 62,` and replace it with `"LogFilter": 95,`
3. Relaunch the application

When you post the log, wrap it with a code block like this to preserve formatting *(makes it much easier to read)*:  
\`\`\`
\< paste log here \>
\`\`\`

**Note:** If you don't want your Windows user account name to be visible in the log, use the find & replace tool in your text editor to replace it with `<USERNAME>` prior to posting it.

**Comment by LL187** [comment]
Ahh, my bad this one should work

[VolumeControl.log](https://github.com/radj307/volume-control/files/13170335/VolumeControl.log)

HH:mm:ss:fff              === Log Initialized @ Mittwoch, 25. Oktober 2023 20:25:26 ===  { Filter: 95 (DEBUG, INFO, WARN, ERROR, FATAL, TRACE) }
22:25:26:203  [DEBUG]     Successfully loaded language config 'C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\de.loc.json'
22:25:26:212  [DEBUG]     Successfully loaded language config 'C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\en.loc.json'
22:25:26:212  [DEBUG]     Successfully loaded language config 'C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\fr.loc.json'
22:25:26:217  [DEBUG]     Successfully loaded language config 'C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\it.loc.json'
22:25:26:222  [DEBUG]     Successfully loaded language config 'C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\pt.loc.json'
22:25:27:015  [DEBUG]     VCSettings initializing...
22:25:27:089  [DEBUG]     .ctor is setting the hook source to '6751152' (Was null)
22:25:27:089  [DEBUG]     Attached a message hook from 'AddMaximizeBugFixHandler'.
22:25:27:092  [DEBUG]     Executable location: 'C:\Users\LL187\AppData\Local\Programs\VolumeControl\VolumeControl.exe'
                          Working directory:   'C:\Users\LL187\AppData\Local\Programs\VolumeControl'
22:25:27:092  [INFO]      Volume Control version 6.3.0 (Installed)
22:25:27:137  [INFO]      Microsoft Windows 11 Pro (64-Bit)
22:25:27:137  [DEBUG]     VCSettings initialization completed.
22:25:27:849  [DEBUG]     Started initializing CoreAudio APIs.
22:25:27:977  [TRACE]     Created AudioSession instance "0:Idle"
22:25:27:982  [TRACE]     Created AudioSession instance "3872:Discord"
22:25:27:982  [TRACE]     Created AudioDevice instance "Lautsprecher (Yeti Classic)"
22:25:28:072  [TRACE]     Created AudioSession instance "0:Idle"
22:25:28:076  [TRACE]     Created AudioSession instance "3872:Discord"
22:25:28:077  [TRACE]     Created AudioDevice instance "Lautsprecher (Steam Streaming Speakers)"
22:25:28:207  [TRACE]     Created AudioSession instance "0:Idle"
22:25:28:211  [TRACE]     Created AudioSession instance "3872:Discord"
22:25:28:211  [TRACE]     Created AudioDevice instance "Realtek Digital Output (Realtek(R) Audio)"
22:25:28:316  [TRACE]     Created AudioSession instance "0:Idle"
22:25:28:320  [TRACE]     Created AudioSession instance "3872:Discord"
22:25:28:321  [TRACE]     Created AudioDevice instance "SAMSUNG (NVIDIA High Definition Audio)"
22:25:28:399  [TRACE]     Created AudioSession instance "24164:VoicemodDesktop"
22:25:28:410  [TRACE]     Created AudioSession instance "0:Idle"
22:25:28:415  [TRACE]     Created AudioSession instance "3872:Discord"
22:25:28:415  [TRACE]     Created AudioDevice instance "Line (Voicemod Virtual Audio Device (WDM))"
22:25:28:578  [ERROR]     An unhandled exception occurred!
                          Sender: 'System.Windows.Threading.Dispatcher' (System.Windows.Threading.Dispatcher)
                          {
                            "Message": "Durch den Aufruf des Konstruktors für Typ "VolumeControl.ViewModels.VolumeControlVM", der den angegebenen Bindungseinschränkungen entspricht, wurde eine Ausnahme ausgelöst."
                            "HResult": "-2146233087"
                            "ExceptionType": "System.Windows.Markup.XamlParseException"
                            "LineNumber": "0"
                            "LinePosition": "0"
                            "KeyContext": ""
                            "UidContext": ""
                            "NameContext": ""
                            "BaseUri": "pack://application:,,,/VolumeControl;component/app.xaml"
                            "Source": "PresentationFramework"
                            "TargetSite": {
                              "Name": "RewrapException"
                              "DeclaringType": "System.Windows.Markup.XamlReader"
                              "Attributes": "PrivateScope, Assembly, Static, HideBySig"
                              "CallingConvention": "Standard"
                            }
                            "Data": {
                              "System.Object": ""
                            }
                            "StackTrace": {
                              [0] at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
                              [1] at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
                              [2] at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
                              [3] at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
                              [4] at System.Windows.Application.LoadBamlStreamWithSyncInfo(Stream stream, ParserContext pc)
                              [5] at System.Windows.Application.LoadComponent(Uri resourceLocator, Boolean bSkipJournaledProperties)
                              [6] at System.Windows.Application.DoStartup()
                              [7] at System.Windows.Application.<.ctor>b__1_0(Object unused)
                              [8] at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
                              [9] at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
                            }
                            "InnerException": {
                              "Message": "Process has exited, so the requested information is not available."
                              "HResult": "-2146233079"
                              "ExceptionType": "System.InvalidOperationException"
                              "Source": "System.Diagnostics.Process"
                              "TargetSite": {
                                "Name": "EnsureState"
                                "DeclaringType": "System.Diagnostics.Process"
                                "Attributes": "PrivateScope, Private, HideBySig"
                                "CallingConvention": "Standard, HasThis"
                              }
                              "StackTrace": {
                                [0] at System.Diagnostics.Process.EnsureState(State state)
                                [1] at System.Diagnostics.Process.get_ProcessName()
                                [2] at VolumeControl.CoreAudio.AudioSession..ctor(AudioDevice owningDevice, AudioSessionControl2 audioSessionControl2)
                                [3] at VolumeControl.CoreAudio.AudioDeviceSessionManager.CreateAndAddSessionIfUnique(AudioSessionControl2 audioSessionControl)
                                [4] at VolumeControl.CoreAudio.AudioDeviceSessionManager..ctor(AudioDevice audioDevice)
                                [5] at VolumeControl.CoreAudio.AudioDevice..ctor(MMDevice mmDevice)
                                [6] at VolumeControl.CoreAudio.AudioDeviceManager.CreateAndAddDeviceIfUnique(MMDevice mmDevice)
                                [7] at VolumeControl.CoreAudio.AudioDeviceManager..ctor(DataFlow deviceDataFlow, MMDeviceEnumerator deviceEnumerator)
                                [8] at VolumeControl.CoreAudio.AudioDeviceManager..ctor(DataFlow deviceDataFlow)
                                [9] at VolumeControl.ViewModels.AudioDeviceManagerVM..ctor()
                                [10] at VolumeControl.ViewModels.VolumeControlVM..ctor()
                                [11] at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
                              }
                            }
                          }

**Comment by radj307** [comment]
Implemented a potential fix in e49053fd1223c5c63769c4fa0d6071f5bb240b45.  

To confirm whether this actually works or not, I created a build artifact of the current repository head. Try the following:
1. Download & extract [this archive](https://github.com/radj307/volume-control/suites/17635389863/artifacts/1008102756)
2. Run the installer
3. Run the program
4. Paste the log file here again

**Comment by LL187** [comment]
This indeed did Fix the issue! Thanks!

Here is the Log File

[VolumeControl.log](https://github.com/radj307/volume-control/files/13180640/VolumeControl.log)

```
HH:mm:ss:fff              === Log Initialized @ Donnerstag, 26. Oktober 2023 17:20:16 ===  { LogFilter: 95 (DEBUG, INFO, WARN, ERROR, FATAL, TRACE) }
19:20:16:627  [TRACE]     [LocalizationHelper] Searching for language config files in "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization"
19:20:16:636  [TRACE]     [LocalizationHelper] Successfully loaded language config "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\de.loc.json"
19:20:16:637  [TRACE]     [LocalizationHelper] Successfully loaded language config "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\en.loc.json"
19:20:16:637  [TRACE]     [LocalizationHelper] Successfully loaded language config "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\fr.loc.json"
19:20:16:638  [TRACE]     [LocalizationHelper] Successfully loaded language config "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\it.loc.json"
19:20:16:639  [TRACE]     [LocalizationHelper] Successfully loaded language config "C:\Users\LL187\AppData\Local\radj307\VolumeControl\Localization\pt.loc.json"
19:20:16:842  [DEBUG]     VCSettings initializing...
19:20:16:918  [DEBUG]     .ctor is setting the hook source to '3605208' (Was null)
19:20:16:918  [DEBUG]     Attached a message hook from 'AddMaximizeBugFixHandler'.
19:20:16:921  [DEBUG]     Executable location: 'C:\Users\LL187\AppData\Local\Programs\VolumeControl\VolumeControl.exe'
                          Working directory:   'C:\Users\LL187\AppData\Local\Programs\VolumeControl'
19:20:16:921  [INFO]      Volume Control version 6.3.0-debug (Installed)
19:20:16:974  [INFO]      Microsoft Windows 11 Pro (64-Bit)
19:20:16:974  [DEBUG]     VCSettings initialization completed.
19:20:17:407  [DEBUG]     Latest version (6.3.0) is newer than the current version.
19:20:17:407  [INFO]      Showing update prompt for new version: 6.3.0
19:20:18:756  [DEBUG]     Started initializing CoreAudio APIs.
19:20:18:870  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Lautsprecher (Yeti Classic)" has 2 associated sessions. Initializing them now...
19:20:18:886  [TRACE]     Created AudioSession instance "0:Idle"
19:20:18:890  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:18:891  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Lautsprecher (Yeti Classic)".
19:20:18:972  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Lautsprecher (Steam Streaming Speakers)" has 2 associated sessions. Initializing them now...
19:20:18:982  [TRACE]     Created AudioSession instance "0:Idle"
19:20:18:986  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:18:986  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Lautsprecher (Steam Streaming Speakers)".
19:20:19:106  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Realtek Digital Output (Realtek(R) Audio)" has 2 associated sessions. Initializing them now...
19:20:19:115  [TRACE]     Created AudioSession instance "0:Idle"
19:20:19:119  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:19:119  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Realtek Digital Output (Realtek(R) Audio)".
19:20:19:222  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "SAMSUNG (NVIDIA High Definition Audio)" has 2 associated sessions. Initializing them now...
19:20:19:232  [TRACE]     Created AudioSession instance "0:Idle"
19:20:19:235  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:19:236  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "SAMSUNG (NVIDIA High Definition Audio)".
19:20:19:313  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Line (Voicemod Virtual Audio Device (WDM))" has 3 associated sessions. Initializing them now...
19:20:19:317  [TRACE]     Created AudioSession instance "24164:VoicemodDesktop"
19:20:19:326  [TRACE]     Created AudioSession instance "0:Idle"
19:20:19:330  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:19:331  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Line (Voicemod Virtual Audio Device (WDM))".
19:20:19:464  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Teufel Headphones (Realtek(R) Audio)" has 12 associated sessions. Initializing them now...
19:20:19:469  [ERROR]     [AudioDeviceSessionManager] An exception occurred while creating an AudioSession instance for session with (ProcessID: 1956, DisplayName: @%SystemRoot%\System32\AudioSrv.Dll,-202, SessionInstanceIdentifier: {0.0.0.00000000}.{eb3396ab-da57-44d5-9f1c-da9774656a70}|#%b{A9EF3FD9-4240-455E-A4D5-F2B3301887B2}|1%b#)!
                          {
                            "Message": "Process has exited, so the requested information is not available."
                            "HResult": "-2146233079"
                            "ExceptionType": "System.InvalidOperationException"
                            "Source": "System.Diagnostics.Process"
                            "TargetSite": {
                              "Name": "EnsureState"
                              "DeclaringType": "System.Diagnostics.Process"
                              "Attributes": "PrivateScope, Private, HideBySig"
                              "CallingConvention": "Standard, HasThis"
                            }
                            "StackTrace": {
                              [0] at System.Diagnostics.Process.EnsureState(State state)
                              [1] at System.Diagnostics.Process.get_ProcessName()
                              [2] at VolumeControl.CoreAudio.AudioSession..ctor(AudioDevice owningDevice, AudioSessionControl2 audioSessionControl2)
                              [3] at VolumeControl.CoreAudio.AudioDeviceSessionManager.CreateAndAddSessionIfUnique(AudioSessionControl2 audioSessionControl)
                            }
                          }
19:20:19:478  [TRACE]     Created AudioSession instance "11076:steam"
19:20:19:478  [TRACE]     Created AudioSession instance "9528:wallpaper64"
19:20:19:481  [TRACE]     Created AudioSession instance "16500:Spotify"
19:20:19:484  [TRACE]     Created AudioSession instance "11280:nvcontainer"
19:20:19:488  [TRACE]     Created AudioSession instance "21048:steamwebhelper"
19:20:19:492  [TRACE]     Created AudioSession instance "13772:Discord"
19:20:19:496  [TRACE]     Created AudioSession instance "24164:VoicemodDesktop"
19:20:19:500  [TRACE]     Created AudioSession instance "24552:firefox"
19:20:19:503  [TRACE]     Created AudioSession instance "24236:VoicemodDesktop"
19:20:19:507  [TRACE]     Created AudioSession instance "6316:NVIDIA Share"
19:20:19:511  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:19:512  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Teufel Headphones (Realtek(R) Audio)".
19:20:19:591  [TRACE]     [AudioDeviceSessionManager] AudioDevice instance "Lautsprecher (Steam Streaming Microphone)" has 2 associated sessions. Initializing them now...
19:20:19:600  [TRACE]     Created AudioSession instance "0:Idle"
19:20:19:603  [TRACE]     Created AudioSession instance "33768:Discord"
19:20:19:604  [TRACE]     [AudioDevice] Successfully created AudioDevice instance "Lautsprecher (Steam Streaming Microphone)".
19:20:19:670  [DEBUG]     Successfully initialized 7 audio devices.
19:20:19:729  [DEBUG]     Successfully initialized 12  audio sessions.
19:20:19:729  [DEBUG]     Finished initializing CoreAudio APIs.
19:20:19:746  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.VolumeUp(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:746  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.VolumeDown(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:746  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.Mute(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.Unmute(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.ToggleMute(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.SelectNext(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.SelectPrevious(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.Deselect(Object sender, HandledEventArgs e).
19:20:19:747  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.SelectDefault(Object sender, HandledEventArgs e).
19:20:19:748  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.Lock(Object sender, HandledEventArgs e).
19:20:19:748  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.Unlock(Object sender, HandledEventArgs e).
19:20:19:748  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioDeviceActions.ToggleLock(Object sender, HandledEventArgs e).
19:20:19:748  [DEBUG]     [ActionLoader] Loaded 12 actions from VolumeControl.HotkeyActions.AudioDeviceActions
19:20:19:748  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.VolumeUp(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.VolumeDown(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.SetVolume(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Target All Sessions" . . . . . (System.Boolean)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
                          + Setting "Volume Level"  . . . . . . . . (System.Int32)
                          + Setting "Mute State"  . . . . . . . . . (System.Boolean)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.Mute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.Unmute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.ToggleMute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Target Override(s)"  . . . . . (VolumeControl.Core.ActionTargetSpecifier)
                          + Setting "Select Target Override(s)" . . (System.Boolean)
19:20:19:749  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.SelectNext(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.SelectPrevious(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.Lock(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.Unlock(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.ToggleLock(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.Deselect(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.AudioSessionActions.ToggleSelected(Object sender, HotkeyPressedEventArgs e).
19:20:19:750  [DEBUG]     [ActionLoader] Loaded 13 actions from VolumeControl.HotkeyActions.AudioSessionActions
19:20:19:751  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ActiveApplicationActions.VolumeUp(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Select Session"  . . . . . . . (System.Boolean)
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:751  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ActiveApplicationActions.VolumeDown(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Select Session"  . . . . . . . (System.Boolean)
                          + Setting "Volume Step Override"  . . . . (System.Int32)
19:20:19:751  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ActiveApplicationActions.Mute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Select Session"  . . . . . . . (System.Boolean)
19:20:19:751  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ActiveApplicationActions.Unmute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Select Session"  . . . . . . . (System.Boolean)
19:20:19:751  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ActiveApplicationActions.ToggleMute(Object sender, HotkeyPressedEventArgs e).
                          + Setting "Select Session"  . . . . . . . (System.Boolean)
19:20:19:751  [DEBUG]     [ActionLoader] Loaded 5 actions from VolumeControl.HotkeyActions.ActiveApplicationActions
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.ToggleSessionNotifications(Object sender, HandledEventArgs e).
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.ToggleDeviceNotifications(Object sender, HandledEventArgs e).
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.BringToForeground(Object sender, HandledEventArgs e).
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.SendToBackground(Object sender, HandledEventArgs e).
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.Minimize(Object sender, HandledEventArgs e).
19:20:19:752  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.ApplicationActions.Unminimize(Object sender, HandledEventArgs e).
19:20:19:752  [DEBUG]     [ActionLoader] Loaded 6 actions from VolumeControl.HotkeyActions.ApplicationActions
19:20:19:753  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.MediaActions.NextTrack(Object sender, HandledEventArgs e).
19:20:19:753  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.MediaActions.PreviousTrack(Object sender, HandledEventArgs e).
19:20:19:753  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.MediaActions.TogglePlayback(Object sender, HandledEventArgs e).
19:20:19:753  [TRACE]     [ActionLoader] Loaded VolumeControl.HotkeyActions.MediaActions.StopPlayback(Object sender, HandledEventArgs e).
19:20:19:753  [DEBUG]     [ActionLoader] Loaded 4 actions from VolumeControl.HotkeyActions.MediaActions
19:20:19:753  [DEBUG]     [ActionLoader] Loaded 40 total actions from 5 action groups.
19:20:19:839  [TRACE]     Disposing of AudioSession instance ""
19:20:20:029  [TRACE]     Enabled Config autosave.

```

**Comment by radj307** [comment]
Glad to hear it's working for you! This fix will be included in the next release version.

**Comment by LL187** [comment]
Appreciate it!

Just real quick, do i assume correctly that i can resume using this build just like normal and use the built in updater when the next release drops, or should i do a full reinstall then?

Thanks a lot!

**Comment by radj307** [comment]
No problem!  
No full reinstall necessary, you can install the next release normally.

**Comment by radj307** [comment]
This fix is now available in [v6.4.0](https://github.com/radj307/volume-control/releases/tag/6.4.0)

**Comment by teadrinker2015** [comment]
If anyone is still experiencing this. Try to use the standalone version, instead of installer one. I'm running it on Windows10 22H2 BTW.
